### PR TITLE
Spelling and grammar fixes

### PR DESCRIPTION
**Pull request by pxlsr**
Just cleaning up the desc's so it is more immersive.

### CODE REVIEW

File: tgstation.dme [hunk 1]
```diff
@@ -6,6 +6,51 @@
 
 // BEGIN_FILE_DIR
 #define FILE_DIR .
+#define FILE_DIR "html"
```
Comment (Jordie0608) [inline tgstation.dme]: You need to go to Build > Preferences and untick 'Automatically set FILE_DIR for sub-directories', then recompile and update this PR.

**Comment by optimumtact** [comment]
Other than what Jordie mentioned

:+1:

File: code/modules/food&drinks/drinks/drinks/drinkingglass.dm [hunk 2]
```diff
@@ -94,7 +94,7 @@
 			if("goldschlager")
 				icon_state = "goldschlagerglass"
 				name = "glass of Goldschlager"
-				desc = "100 proof that teen girls will drink anything with gold in it."
```
Comment (Cheridan) [inline code/modules/food&drinks/drinks/drinks/drinkingglass.dm]: I thought this was pun on the usage of the word "proof" in drinks to denote percentage of alcohol. Goldschlager is traditionally between 87-107 proof so it's pretty accurate.

File: code/modules/food&drinks/drinks/drinks/drinkingglass.dm [hunk 3]
```diff
@@ -450,7 +450,7 @@
 			if("vanillapudding")
```
Comment (Cheridan) [inline code/modules/food&drinks/drinks/drinks/drinkingglass.dm]: >Vanilla pudding is on fire

SIR I HAVE A PROBLEM WITH MY DESSERT OH GOD 

:bangbang: :fire:
:scream: :custard:

File: code/modules/food&drinks/drinks/drinks/drinkingglass.dm [hunk 4]
```diff
@@ -450,7 +450,7 @@
...
 				icon_state = "vanillapudding"
 				name = "Vanilla Pudding"
-				desc = "Whoah, this thing is on FIRE"
+				desc = "Whoah, this thing is on FIRE!"
```
Comment (Cheridan) [inline code/modules/food&drinks/drinks/drinks/drinkingglass.dm]: >Vanilla pudding is on fire

SIR I HAVE A PROBLEM WITH MY DESSERT

:bangbang: :fire:
:scream: :custard:

**Pull request closed without merging**